storage-proofs/src/circuit/por.rs

```diff
@@ -336,19 +400,28 @@ mod tests {
 
     #[test]
     fn test_por_input_circuit_with_bls12_381_pedersen() {
-        test_por_input_circuit_with_bls12_381::<PedersenHasher>(4149);
+        test_por_input_circuit_with_bls12_381::<PedersenHasher, PedersenHasher>(5531);
```

Comment:
The reason is because of the `ALPHA_TREE_HEIGHT`. An alpha tree height of 3 would require at least 8 leaves, add an additional layer for the required beta hasher's layer and you get a minimum number of leaves of 16.

Making `ALPHA_TREE_HEIGHT` a const generic (or constructor parameter) on `HybridMerkleTree` instead of a crate wide constant would mean that each uniquely parameterized Hybrid Merkle Tree would create a unique `PoRCircuit` because that circuit depends on the the number of alpha vs beta hashes.

Reply:
That's right. When beta height varies by layer, the `PoRCircuit`s will also do so.